Remove note and improve Airflow Tutorial (#3548)
airbytehq/airbyte: docs/tutorials/using-the-airflow-airbyte-operator.md

---
description: Start triggering Airbyte jobs with Apache Airflow in minutes
---

# Using Apache Airflow with the Airbyte Operator

> Airbyte is an official community provider in Apache Airflow project.
> The Airbyte operator allows you to trigger synchronization jobs in Apache Airflow. 
> This tutorial explains how to configure your Airflow DAG to do so.

Reviewer: ```suggestion
Airbyte is an official community provider for the Apache Airflow project. The Airbyte operator allows you to trigger synchronization jobs in Apache Airflow, and this tutorial will walk through configuring your Airflow DAG to do so.
```

For the purposes of this tutorial, set your Connection's **sync frequency** to **manual**. Airflow will be responsible for manually triggering the Airbyte job.

### **Start Apache Airflow**

> If you don't have an Airflow instance, we recommend following this [Quick Start Apache Airflow Tutorial](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html) to set one up.

Reviewer: ```suggestion
If you don't have an Airflow instance, we recommend following this [guide](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html) to set one up.
```

> You need to install additionally the `apache-airflow-providers-airbyte` package to use Airbyte Operator on Apache Airflow.

Reviewer: ```suggestion
Additionally, you will need to install the `apache-airflow-providers-airbyte` package to use Airbyte Operator on Apache Airflow.
```

> You can read more about [here](https://airflow.apache.org/docs/apache-airflow-providers-airbyte/stable/index.html)

Reviewer: ```suggestion
You can read more about it [here](https://airflow.apache.org/docs/apache-airflow-providers-airbyte/stable/index.html)
```
